src/main/java/com/github/manolo8/darkbot/utils/login/LoginUtils.java

```diff
@@ -113,22 +114,41 @@ public static void usernameLogin(LoginData loginData) {
         try {
             usernameLogin(loginData, "www");
         } catch (Exception e) {
-            usernameLogin(loginData, "lp");
+            try {
+                usernameLogin(loginData, "lp");
+            } catch (IOException ex) {
+                throw new RuntimeException(ex);
```

Comment:
```suggestion
                throw new LoginException(ex);
```
Let's reuse our exception properly (also keep in mind you'll need to add a constructor with a cause exception to pass to super(ex) )